back/src/core/sockets/index.ts

```diff
@@ -0,0 +1,52 @@
+import SocketIOClient, { Socket } from 'socket.io';
+import { Action, SocketInfo } from 'dals/messages';
+import {
+  processInputMessage,
+  InputMessageTypes,
+  processOutputMessageCollection,
+} from 'pods/messages';
+
+export const sockets = async (socket: Socket, io: SocketIOClient.Socket) => {
+  // WATCH OUT !! Reconnect is not implemented
+  // In the connection input processing, we should
+  // check if connectionId matches with userId and RoomId
+  // if not reject, if it's accepted send connection
+  // reestablished
+  const { user, room, isMaster } = socket.handshake.query;
```

Comment:
If this functionality is in core, why we have here this code about user, room, etc¿

Reply:
@brauliodiez ?